### PR TITLE
added Position2DInt.msg

### PR DESCRIPTION
**Pull request by yathartha3**
---

## Basic Info
Adding Position2DInt message, because porting base_local_planner requires this message.

---

## Description of contribution in a few bullet points

---

## Future work that may be required in bullet points

---

I'd like to request maintainer: <blank> to review this PR.

### CODE REVIEW

File: nav2_msgs/msg/Position2DInt.msg [hunk 1]
```diff
@@ -0,0 +1,2 @@
+int64 x
+int64 y
```
Comment (orduno) [inline nav2_msgs/msg/Position2DInt.msg]: Missing a new line.

**Comment by SteveMacenski** [comment]
@yathartha3 any update? or is this not needed anymore?

**Comment by mhpanah** [comment]
@yathartha3 Why this is needed in `base_local_planner`?

**Comment by yathartha3** [comment]
@mhpanah It was used by TEB. However I am closing this PR since I will be submitting the PR for local planner interface.

**Pull request closed without merging**